hw3/02-charts.html

```diff
@@ -14,7 +14,7 @@
   <body>
     <main>
       <h1>Exercise 02 - Charts</h1>
```

Comment:
I think you might need to give main a white background and wrap the h1 title and the chart within to get the look like the reference photo.  Overall, looking great. I like how you used random colors to populate the chart.